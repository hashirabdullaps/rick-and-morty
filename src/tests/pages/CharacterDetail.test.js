import React from 'react';
import renderer from 'react-test-renderer';
import CharacterDetailPage from '../../pages/character-detail/CharacterDetail';
import { MockedProvider } from '@apollo/client/testing';
import { BrowserRouter } from "react-router-dom";

const mocks = [];
test('should test Character Detail component', () => {
    const component = renderer.create(
        <MockedProvider mocks={mocks} addTypename={false}>
            <BrowserRouter>
                <CharacterDetailPage />
            </BrowserRouter>
        </MockedProvider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});