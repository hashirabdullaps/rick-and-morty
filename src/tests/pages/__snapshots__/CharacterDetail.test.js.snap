// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character Detail component should test Character Detail component 1`] = `
<div
  style={
    Object {
      "height": "50vh",
    }
  }
>
  <div
    className="ant-spin ant-spin-spinning spinner-loading"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
  </div>
</div>
`;
